plugins/mcreator-localization/lang/texts.properties

```diff
@@ -369,6 +371,7 @@ dialog.preferences.page_plugins=Manage plugins
 dialog.preferences.page_themes=UI themes
 dialog.preferences.page_procedure_templates=Procedure templates
 dialog.preferences.page_ai_builder_templates=AI builder templates
+dialog.preferences.page_cmd_builder_templates=Command arguments templates
```

Comment:
This should be something shorter so the preferences list does not get too wide

Reply:
We can add a disabled **Templates** label to the preferences list before initializing any template panels to indicate that any following tabs offer editing diverse templates, which will enable this string to be shortened to _Command arguments_.

Reply:
It is not the string here, but how it looks in UI and moving location where templates is added does not change that

Reply:
> This should be something shorter so the preferences list does not get too wide

What do you suggest? We can't write something more precise than `Command arguments`.

Reply:
Cmd. argument templates maybe? Not sure, but right now it is too wide IMO :D

Reply:
Only 4 chars shorter, is that crucial?

Reply:
Open MCreator and you can see it is really wide compared to the whole UI, also if you have idea for shorter variant, it is very welcome, that was just my initial guess

Reply:
Not ideal, but maybe "Command templates" or even "Cmd. templates" could work

Reply:
```suggestion
dialog.preferences.page_cmd_builder_templates=Command templates
```